docs/dev/nightly_data_builds.rst

```diff
@@ -292,37 +284,58 @@ that are available:
 
 .. code::
 
-   gsutil ls gs://nightly-build-outputs.catalyst.coop
+   gsutil ls -lh gs://nightly-build-outputs.catalyst.coop
 
-You should see a list of directories with the naming convention
-``<git commit SHA>-<git branch>``.
+You should see a list of directories with build IDs that have a naming convention:
+``<YYYY-MM-DD-HHMM>-<short git commit SHA>-<git branch>``.
 
-To see what the outputs are for a given nightly build, you can use ``gsutil``
-like this:
+To see what the outputs are for a given nightly build, you can use ``gsutil`` like this:
 
 .. code::
 
-   gsutil ls -l gs://nightly-build-outputs.catalyst.coop/<build name of interest>
-
-      1152800  2022-11-22T12:51:02Z  gs://nightly-build-outputs.catalyst.coop/<build name of interest>/pudl-etl.log
-                                 gs://nightly-build-outputs.catalyst.coop/<build name of interest>/parquet/
-                                 gs://nightly-build-outputs.catalyst.coop/<build name of interest>/pudl_out/
-                                 gs://nightly-build-outputs.catalyst.coop/<build name of interest>/sqlite/
-   TOTAL: 1 objects, 1152800 bytes (1.1 MiB)
+    gsutil ls -lh gs://nightly-build-outputs.catalyst.coop/2024-01-03-0605-e9a91be-dev/
```

Comment:
I'd love if we could shorten this URL. I can never remember what it's supposed to be exactly.  Maybe just `gs://builds.catalyst.coop`?

Reply:
I'm down, can we just... do it?

Reply:
I'm down to shorten it. Maybe `gss://pudl-builds.catalyst.coop`? Adding `pudl` might be redundant because everything in the `catalyst-cooperative-pudl` project should be related to `pudl`

Reply:
can I advocate for `gss://nightly.catalyst.coop`? I think that makes it clearer that it's the nightly updated data, personally.

Reply:
But it's not just the nightly builds -- it's also the stable builds and the ad-hoc builds. Any data build will deposit its outputs in here.

Reply:
Okay, I've updated this all to point at `gs://builds.catalyst.coop` and have copied over all of our existing build outputs to a newly created bucket, so hopefully it works.